compiler/luci-value-py-test/conftest.py

```diff
@@ -0,0 +1,74 @@
+import re
```

Comment:
This file is for pytest configuration.

https://docs.pytest.org/en/7.1.x/example/parametrize.html#parametrizing-tests